### PR TITLE
Update to 3.2.3

### PR DESCRIPTION
**Pull request by benjamreis**
[v8.3-incoming scratch build](https://koji.xcp-ng.org/taskinfo?taskID=63019)

### CODE REVIEW

**Review by Nambrok — APPROVED** [review]

File: SOURCES/0002-feat-drivers-add-CephFS-and-GlusterFS-drivers.patch [hunk 1]
```diff
@@ -207,8 +207,8 @@ index 0000000..d797490
 +            try:
 +                self.mount()
 +                os.symlink(self.linkpath, self.path)
-+            except CephFSException, exc:
-+                raise SR.SROSError(12, exc.errstr)
++            except CephFSException as exc:
```
Comment (stormi) [inline SOURCES/0002-feat-drivers-add-CephFS-and-GlusterFS-drivers.patch:210]: It feels a bit strange to make this change in the first patch, when there are other patches that will be applied later to alter it for python3 support, IPv6 support, etc.

To be discussed when @Wescoeur is back?

**Review by stormi — APPROVED** [review]
LGTM overall, with one question: no other changes brought to drivers since 3.2.0 need to be also applied to our own drivers?

File: SOURCES/0002-feat-drivers-add-CephFS-and-GlusterFS-drivers.patch (hunk 1 above)
Comment (benjamreis) [reply SOURCES/0002-feat-drivers-add-CephFS-and-GlusterFS-drivers.patch:210]: Yeah I think we need to re-do our patches to have less.

**Comment by benjamreis** [comment]
> LGTM overall, with one question: no other changes brought to drivers since 3.2.0 need to be also applied to our own drivers?

I looked at the the commits and from what I understood - no. maybe @Nambrok can have a look and confirm before we merge?

**Pull request merged**